themes/theme-boilerplate/views/default.master.tpl

```diff
@@ -34,14 +34,14 @@
             </div>
         {/if}
     {/if}
-    {categories_link format=$linkFormat}
-    {discussions_link format=$linkFormat}
-    {if $DataDrivenTitleBar}
-        {knowledge_link format=$linkFormat}
-    {/if}
+    
     {if !$DataDrivenTitleBar}
-        {activity_link format=$linkFormat}
```

Comment:
Put activity back on top.

Reply:
create capture for navLinks if !DataDrivenTitleBar